hub-js/src/index.ts

```diff
@@ -68,4 +69,6 @@ function setupHttpServer(
   return server;
 }
 
-main();
+(async () => {
```

Comment:
was this a bug or is this a new requirement for main to be async?

Reply:
This is something required by new version of Apollo dependency. The Apollo server needs to be started, and this is asynchronous call.

```
const apolloServer = new ApolloServer({ schema, context: { driver } });
await apolloServer.start(); // <-- this line
apolloServer.applyMiddleware({ app });
```